### PR TITLE
VisionX Implementation Guide

### PR DESCRIPTION
**Pull request by kashifzeeshan**
VisionX Implementation Guide

## What Changed?

Please list, at a high level, what changed in your branch. If you changed content, include the product, section, and version as applicable. **Please remove the user instructions including the examples before merging the PR.**

**Examples**

- Fixed typo in EPAS 13 epas_inst_linux guide
- Added more detail to ARK 3.5 getting started guide introduction
- Fixed broken link on `/supported-open-source/pgbackrest/05-retention_policy/` page

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: advocacy_docs/partner_docs/VisionXGuide/02-PartnerInformation.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+description: 'Details for VisionX'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | SIB Visions       |
+| **Partner Product**   |  VisionX       |
+| **Web Site**      |    https://visionx.sibvisions.com/   |
+| **Version**   |  VisionX 5.6.925      |
+| **Product Description**      |  VisionX is a low-code platform for creating web and mobile applications extremely quickly and easily. VisionX and its Oracle Forms Migration Extension are the most efficient solution for semi-automated migration of Oracle Forms and Oracle APEX into modern web and mobile applications. VisionX, together with EDB Postgres Advanced Server migration, can help provide an exit strategy from Oracle to Postgres.    |
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/02-PartnerInformation.mdx:12]: Solution summary:
VisionX is a low-code platform for creating web and mobile applications. VisionX and its Oracle Forms Migration Extension are the an efficient solution for semi-automated migration of Oracle Forms and Oracle APEX into modern web and mobile applications. VisionX, together with an EDB Postgres Advanced Server migration, can help provide an exit strategy from Oracle to Postgres.

File: advocacy_docs/partner_docs/VisionXGuide/03-SolutionSummary.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+VisionX is the most flexible and independent low-code platform available, enabling both business users and professional developers to visually develop web, desktop and native mobile applications extremely fast. These can be very simple applications that replace paper processes or Excel sheets, easy-to-use forms on ERP systems, dashboards, mobile apps, and even  highly complex billing applications, customer portals or trading systems.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/03-SolutionSummary.mdx:5]: change all to
VisionX is a flexible and independent low-code platform, enabling both business users and professional developers to visually develop web, desktop and native mobile applications quickly. These can be very simple applications that replace paper processes or Excel sheets, easy-to-use forms on ERP systems, dashboards, mobile apps, and even  highly complex billing applications, customer portals or trading systems.

File: advocacy_docs/partner_docs/VisionXGuide/03-SolutionSummary.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+VisionX is the most flexible and independent low-code platform available, enabling both business users and professional developers to visually develop web, desktop and native mobile applications extremely fast. These can be very simple applications that replace paper processes or Excel sheets, easy-to-use forms on ERP systems, dashboards, mobile apps, and even  highly complex billing applications, customer portals or trading systems. 
+
+The VisionX low-code platform supports major databases, such as Postgres, EDB, Oracle, MS SQL, MySQL and SAP HANA. VisionX is automatically bundled with a Postgres database and is a low-code platform for Postgres.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/03-SolutionSummary.mdx:7]: change to
The VisionX low-code platform supports major databases, such as Postgres, EDB Postgres Advanced Server and EDB Postgres Extended Server. VisionX is automatically bundled with a Postgres database and is a low-code platform for Postgres.

File: advocacy_docs/partner_docs/VisionXGuide/03-SolutionSummary.mdx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+VisionX and its Oracle Forms Migration Extension are the most efficient solution for semi-automated migration of Oracle Forms and Oracle APEX into modern web and mobile applications. Together with an Oracle database to EDB Postgres migration, it provides a fast, smooth, and reliable Oracle exit strategy.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/03-SolutionSummary.mdx:9]: Change to
VisionX and its Oracle Forms Migration Extension are the an efficient solution for semi-automated migration of Oracle Forms and Oracle APEX into modern web and mobile applications. Together with an Oracle database to an EDB Postgres Advanced Server migration, it provides a fast, smooth, and reliable Oracle exit strategy.

File: advocacy_docs/partner_docs/VisionXGuide/04-ConfiguringTheIntegration.mdx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring VisionX'
+---
+
+Implementing VisionX with EDB Postgres Advanced Server and EDB Postgres Extended Server requires the following components:
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/04-ConfiguringTheIntegration.mdx:6]: Change "and" to "or" between EDB DB servers names.

File: advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx [hunk 6]
```diff
@@ -0,0 +1,485 @@
+---
+title: 'Using VisionX'
+description: 'Walkthroughs of multiple VisionX usage scenarios'
+---
+
+This section shows following VisionX usage scenarios with EDB Postgres Advanced Server and EDB Postgres Extended Server.
+
+1. FMB-based migration of an Oracle Forms application to an EDB Postgres Advanced Server / VisionX Low Code application
+  - Part 1: FMB based application migration
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx:9]: Indent all bullets

File: advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx [hunk 7]
```diff
@@ -0,0 +1,485 @@
...
+  - Part 2: Oracle database to EDB Postgres Advanced Server migration
+  - Part 3: Switch database connection to EDB Postgres Advanced Server
+2. Data model-based migration of an Oracle Forms / APEX application to an EDB Postgres / VisionX Low Code application
+  - Part 1: Data model based application migration
+  - Part 2: Oracle database to EDB Postgres Advanced Server migration
+  - Part 3: Switch database connection to EDB Postgres Advanced Server
+3. Create a VisionX application that uses an EDB Postgres Advanced Server / EDB Postgres Extended instance as a data source
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx:16]: use "or" and not "/" between EDB DB server names.   3 places.

File: advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx [hunk 8]
```diff
@@ -0,0 +1,485 @@
...
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration17.png">
+</p>
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration18.png">
+</p>
+
+   If you change the code outside of VisionX in Eclipse, the changes are pushed back into the VisionX visual development environment in real time! This enables 100% unlimited app development for Citizen Developers / Business Users and Pro Developers.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx:136]: Change "!" to "."
development environment in real time.

remove 100%

File: advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx [hunk 9]
```diff
@@ -0,0 +1,485 @@
...
+
+10. Open the screen in the VisionX Live Preview and change some of the data. The data will be updated in the Oracle Database.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration19.png">
+</p>
+
+### Oracle database to EDB Postgres Advanced Server migration
+
+In the second step, the Oracle database is migrated to EDB Postgres Advanced Server/EDB Postgres Extended Server using the EDB's MTK migration tool.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx:146]: MTK full name. (migration tool kit, I think)

File: advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx [hunk 10]
```diff
@@ -0,0 +1,485 @@
...
+
+## Data model-based migration of an Oracle Forms / APEX application to an EDB Postgres Advanced Server / VisionX Low Code application
+
+This example shows how an Oracle Forms / APEX application is migrated to a VisionX low-code application using the existing data model. The first step is to migrate the application. In the second step, the Oracle database is migrated to EDB Postgres Advanced Server/EDB Postgres Extended using the usual tools such as EDBs MTK.
+
+Then the migrated application is connected to EDB Postgres Advanced Server/EDB Postgres Extended Server.
+
+### Data model based application migration
+
+Steps 1 to 5 are the same as for an FMB-based application migration.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx:168]: Change
Steps 1 to 5 are the same as for an FMB-based application migration.
to
Follow Steps 1 to 5 under section FMB-based application migration. and then go to step 1.

Update steps to start with 1, then 2, and so on.

File: advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx [hunk 11]
```diff
@@ -0,0 +1,485 @@
...
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration37.png">
+</p>
+
+## Create a VisionX application that uses an EDB Postgres Advanced Server / EDB Postgres Extended instance as a data source
+
+### Create a screen that connects to an existing table from an EDB Postgres Advanced Server / EDB Postgres Extended instance
+
+1. Data setup on Postgres Advanced Server 13 instance.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx:288]: For this example data was setup in an EDB Postgres Advanced Server 13 instance.
That is line 1.

File: advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx [hunk 12]
```diff
@@ -0,0 +1,485 @@
...
+
+```
+
+2. Open VisionX and create an application. The “Create a new Application” screen will be opened.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/ImportExport24.png">
+</p>
+
+3. At the “Choose Database” step, enter the connection information for the EDB Postgres Advanced Server 13 instance.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx:342]: delete "13". don't need the version anymore.

File: advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx [hunk 13]
```diff
@@ -0,0 +1,485 @@
...
+  <img  alt="Create a new Application" src="Images/FMBMigration8.png">
+</p>
+
+8. At the “Choose Module” step, select the FMB file.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration9.png">
+</p>
+
+   In this example, VisionX analyzes the orders.fmb file and shows all included windows/canvases for the selection.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx:79]: indent

File: advocacy_docs/partner_docs/VisionXGuide/06-CertificationEnvironment.mdx [hunk 14]
```diff
@@ -0,0 +1,19 @@
+---
+title: 'Certification Environment'
+description: 'Overview of the certification environment used in the certification of VisionX'
+---
+
+|  &nbsp;    | &nbsp; |
+| ----------- | ----------- |
+| **Certification Test Date**      | March 31, 2022       |
+| **EDB Postgres Advanced Server**      | 14/13/12/11       |
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/06-CertificationEnvironment.mdx:9]: change version order for all.  Should go lowest to high.

File: advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx [hunk 15]
```diff
@@ -0,0 +1,487 @@
...
+
+5. Open the screen in the SIB Visions VisionX Live Preview and change some of the data. The data will be updated in the Oracle Database.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration19.png">
+</p>
+
+### Oracle database to EDB Postgres Advanced Server migration
+
+In the second step, the Oracle database is migrated to EDB Postgres Advanced Server/EDB Postgres Extended Server using the EDB's Migration Toolkit.
```
Comment (scottatchue) [inline advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx]: Looks like it got missed.  We need to update MTK in all places.  6 I think.  Should use full name.  Migration Toolkit.

**Review by josh-heyer — APPROVED** [review]

**Pull request merged**